### PR TITLE
fbo_oci_backup_inspector

### PR DESCRIPTION
**Pull request by Olygo**
commit = 1

### CODE REVIEW

**Comment by Olygo** [comment]
I have read the CLA Document and I hereby sign the CLA

File: cloud-infrastructure/compute-including-hpc/compute/python_scripts/OCI-BackupInspector/OCI-BackupInspector.py
Comment (AlexanderHodicke) [inline cloud-infrastructure/compute-including-hpc/compute/python_scripts/OCI-BackupInspector/OCI-BackupInspector.py:1]: @oheimburger  Please review

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]
Code review

Rest is good

File: cloud-infrastructure/compute-including-hpc/compute-software/python_scripts/OCI-BackupInspector/modules/utils.py [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+        volume_group_name = data.get('volume_group_name', '').replace(' - ', '')
+        volume_group_id = data.get('volume_group_id', '').replace(' - ', '')
+        is_volume_group_backup = data.get('is_volume_group_backup', '').replace(' - ', '')
+        attached_instance = data.get('attached_instance', '').replace(' - ', '')
+        attached_instance_id = data.get('attached_instance_id', '').replace(' - ', '')
+
+        with open(csv_report, mode='a', newline='') as csvfile:
+            writer = csv.DictWriter(
+                csvfile,
+                fieldnames=[
```
Comment (oheimburger) [inline cloud-infrastructure/compute-including-hpc/compute-software/python_scripts/OCI-BackupInspector/modules/utils.py:262]: Duplication of code. Consolidate.

**Pull request closed without merging**